includes/definitions/quantum.yaml

```diff
@@ -0,0 +1,13 @@
+os: quantum
+type: server
+text: Quantum
+mib_dir: adic
+ifXmcbc: true
+ifname: true
+poller_modules:
```

Comment:
Disabling poller modules is not what you want.  They should be skipped if there is no discovered data for the devices anyway.